nexus/src/db/datastore/vpc.rs

```diff
@@ -675,4 +705,66 @@ impl DataStore {
                 )
             })
     }
+
+    /// Identify all subnets in use by each VpcSubnet
+    pub async fn resolve_subnets_to_ips<T: IntoIterator<Item = Name>>(
+        &self,
+        vpc: &Vpc,
+        subnet_names: T,
+    ) -> Result<HashMap<Name, Vec<IpNetwork>>, Error> {
```

Comment:
Out of curiosity, why a hash map rather than a B-tree? `Name` is sortable, and they should all be unique since they're scoped to the VPC.

Reply:
Nope, no good reason. Changed to use `BTreeMap` in cbedb56b.